proposals/0031-hlsl-vector-matrix-operations.md

```diff
@@ -530,7 +533,9 @@ Mul(MatrixRefImpl<MatrixBufferTy, MatrixDT, MatrixM, MatrixK, MatrixLayout,
 ## Function: MulAdd
 
 The `dx::linalg::MulAdd` function behaves as `dx::linalg::Mul`, but also adds a
-bias vector (loaded from memory) to the result.
+bias vector (loaded from memory) to the result. Similar to the the matrix
+operand, the memory backing the bias vector must be read-only, and therefore an
+object of `VectorRef` type (and not `RWVectorRef` type).
```

Comment:
As above except without the ability to specify RWVectorRef here, the type is not needed at all.

Reply:
+1